### DISCUSSION TITLE
Error running wflow in fews

### DISCUSSION THREAD

**Discussion by Radheika2424** [Deltares/Wflow.jl#483] [Q&A]
![IMG-20241012-WA0021](https://github.com/user-attachments/assets/27bdd716-aa14-4e3f-ac54-200e24908fdb)
![IMG-20241012-WA0020](https://github.com/user-attachments/assets/7621eeec-f36b-4537-a3fa-b71f5c85a09e)
![IMG-20241012-WA0019](https://github.com/user-attachments/assets/ebe9ccd5-c6a3-4bd0-bfa3-11f08fe4f540)
![IMG-20241012-WA0018](https://github.com/user-attachments/assets/5ce86eac-429b-4a70-af66-5d10939cdcfd)

Anyone can help me with this issue

**Comment by verseve** [comment]
The grid dimensions of the input to the wflow model are not equal to the wflow model grid definition (defined by the staticmaps). I would recommend to check the files that FEWS exports to wflow, and compare with a staticmap of the wflow model if this is indeed the case. If this is the case, the grid definition should be changed in the FEWS config.

I noticed the wflow Python version is used. Please note that this repository is about the wflow Julia version. There will be no further developments in the Python wflow framework, developments continue in the wflow Julia version.
